tests/conftest.py

```diff
@@ -80,6 +85,30 @@ def cert_and_key_paths():
         yield cert_temp_path, key_temp_path
 
 
+class CAWithPKEncryption(trustme.CA):
+    """Implementation of trustme.CA() that emits private keys
+    that are encrypted with a password.
+    """
+
+    @property
+    def private_key_pem(self):
+        return trustme.Blob(
+            self._private_key.private_bytes(
+                Encoding.PEM,
+                PrivateFormat.TraditionalOpenSSL,
+                BestAvailableEncryption(password=b"password"),
+            )
+        )
+
+
+@pytest.fixture
+def cert_and_encrypted_key_paths():
+    ca = CAWithPKEncryption()
+    ca.issue_cert("example.org")
+    with ca.cert_pem.tempfile() as cert_temp_path, ca.private_key_pem.tempfile() as key_temp_path:
+        yield cert_temp_path, key_temp_path
```

Comment:
Could we use something like this?...

```python
@pytest.fixture
def example_cert():
    ca = CAWithPKEncryption()
    ca.issue_cert("example.org")
    return ca

@pytest.fixture 
def cert_pem_file(example_cert):
with example_cert.cert_pem.tempfile() as cert_temp_path:
    yield cert_temp_path

@pytest.fixture 
def cert_private_key_file(example_cert):
with example_cert.private_key_pem.tempfile() as key_temp_path:
    yield key_temp_path
```

In order to allow us to pass fixtures as `cert_pem_file, cert_private_key_file` rather than unpacking a two-tuple in the test body?